airbyte-integrations/connectors/source-amazon-ads/streams/attribution_report_performance_adgroup.pk.json

```diff
@@ -0,0 +1 @@
+["adGroupId"]
```

Comment:
The key of a collection must be a unique identifier of documents, and it can be a compound key (multiple properties form a key together).

In this case, `adGroupId` is not a unique identifier of attribution reports. My _guess_ is that these properties are necessary to form a key for attribution reports:

[The Model](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/schemas/attribution_report.py)

Compound Key:
- date
- campaignId
- adGroupId
- publisher
- advertiser

And based on which attribution report we are considering, additional properties might become part of the key, for example, for performance attribution reports, I think these should also be included:
- creativeId

And for product reports:
- productName

Some documentation: https://advertising.amazon.com/API/docs/en-us/amazon-attribution-prod-3p

Reply:
Added the other attribute as keys and additional in product and creative.